open_spiel/games/goofspiel.cc

```diff
@@ -577,7 +577,7 @@ void GoofspielState::ObservationTensor(Player player,
 
   // Current point card.
   for (int i = 0; i < num_cards_; ++i) {
-    values->push_back(i == point_card_index_ ? 1.0 : 0.0);
+    values->push_back(i == point_card_sequence_.back() ? 1.0 : 0.0);
```

Comment:
This is not safe if point_card_sequence is empty, which it will be at the start of the game.

Reply:
Isn't the numbers of the cards are from 1 to 13 in the point_card_sequence? This would be a misinformation regarding which card has been played on the current trick in terms of ObervationTensor?

Reply:
@elkhrt You are right about it being unsafe. I addressed this issue and the other issue raised by @Batom3 in the subsequent commit.